classes/PodsMeta.php

```diff
@@ -985,6 +985,8 @@ public function meta_post ( $post, $metabox ) {
 			self::$current_pod->fetch( $id );
 
         $pod = self::$current_pod;
+	// filter the pods metas to display
+	$metabox[ 'args' ][ 'group' ][ 'fields' ] = apply_filters( 'pods_meta_' . __FUNCTION__ . '_fields', $metabox[ 'args' ][ 'group' ][ 'fields' ], $id );
```

Comment:
Can you rename this to pods_meta_post_fields instead? And be sure to pass in $post, $metabox, $pod too

Also needs phpdoc for filter if possible https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters